### DISCUSSION TITLE
Delete __Secure-next-auth.session-token.0 & __Secure-next-auth.session-token.1

### DISCUSSION THREAD

**Discussion by D-Thrane** [nextauthjs/next-auth#3679] [Help]
Is there any easy way to get next-auth to delete the cookies __Secure-next-auth.session-token.0 & __Secure-next-auth.session-token.1 without doing a logout.

**Comment by balazsorban44** [comment]
No, and this is intended. The session is created on the behalf of the user, so unless expired, only they should be allowed remove it.

**Comment by EduardoFromTheFuture** [comment]
What are the next-auth.session-token.0 and next-auth.session-token.1?
In other words, why are there two?

I see that only some users get 2 of these tokens, while most of the users in my application only get one.

I seem to be having problems because of this, that's why I'm investigating, but so far I found nothing useful.

I'm guessing that maybe one of them is some kind of refresh token?
